### PR TITLE
Fixes examine on mindless humans

### PR DESCRIPTION
**Pull request by AnturK**

### CODE REVIEW

**Comment by RaveRadbury** [comment]
Hi, this is from my recent PR. Can you explain what the problem is? I thought the src. was to prevent you from examining yourself, also this only applies in regards to the spiritual quirk?

**Comment by AnturK** [comment]
src is unnecessary here, it would only matter if there was variable with the same name in local scope to differentiate between them. The real issue here is you didn't check if the mob actually has mind before trying to grab it's variable.

**Comment by RaveRadbury** [comment]
Ah, I see. At the time I was concerned with getting the perk from a Chaplain that had SSD'ed or something like that. Didn't even think about the holy trait being on the mind. Thank you for fixing this!

File: code/modules/mob/living/carbon/human/examine.dm [hunk 1]
```diff
@@ -272,7 +272,7 @@
 
 			msg += "</span>"
 
-			if(HAS_TRAIT(user, TRAIT_SPIRITUAL) && src.mind.isholy)
+			if(HAS_TRAIT(user, TRAIT_SPIRITUAL) && mind && mind.isholy)
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/mob/living/carbon/human/examine.dm]:
```suggestion
			if(HAS_TRAIT(user, TRAIT_SPIRITUAL) && mind?.isholy)
```

**Pull request merged**